### ISSUE TITLE
batchnorm calibration的代码？

### ISSUE THREAD

**Issue by ShuangtongLi** [ModelTC/MQBench#202]
作者你好，非常感谢你能够将这么好的量化工具公开。我看你的论文里面有关于batchnorm calibration的结果（先量化，然后根据量化的模型调整batchnorm的scale和variance）。想问一下这个部分在mqbench里面有实现吗，有没有示例代码可以复现一下结果呢？万分感谢！

**Comment by Tracin** [comment]
`model.train()`但是不`loss.backward()`或`optimizer.step()`即可
实际上不是特别好用
